webpack/assets/javascripts/react_app/components/common/ClipboardCopy/index.js

```diff
@@ -11,16 +11,33 @@ const ClipboardCopy = ({
   buttonText,
   textareaProps,
   buttonProps,
+  hideTextarea,
+  withCopyIcon,
 }) => {
   const [text, setText] = useState(defaultText);
 
+  let clipboardClass = 'clipboard-copy';
+  let iconMarginLeft = '5px';
+
+  if (hideTextarea === true) {
+    clipboardClass = 'clipboard-copy-simple';
+  }
+
+  if (buttonText === '') {
+    iconMarginLeft = '0px';
+  }
```

Comment:
Just to reduce some space I guess we could use ternary operators:
```suggestion
  const clipboardClass = hideTextarea ? 'clipboard-copy-simple' : 'clipboard-copy';
  const iconMarginLeft = buttonText === '' ? '0px' : '5px';
```

Reply:
thanks.